File: atcoder.jp/contests/jsc2021/tasks/jsc2021_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

int div_ceil(int a, int b) { return (a + b - 1) / b; }

int main() {
  int X, Y, Z;
  cin >> X >> Y >> Z;

  int ans = div_ceil(Y * Z, X) - 1;
  cout << ans << endl;
}

File: atcoder.jp/contests/jsc2021/tasks/jsc2021_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const divCeil = (a: number, b: number) => Math.trunc((a + b - 1) / b);

const main = () => {
  const [X, Y, Z] = input().split(" ").map(Number);

  const ans = divCeil(Y * Z, X) - 1;
  console.log(ans);
};

main();
